pkgs/by-name/sc/scalar/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  electron,
+  dpkg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scalar";
+  version = "0.1.188";
+
+  src = fetchurl {
+    url = "https://download.scalar.com/linux/deb/x64";
+    name = "scalar-app-${version}amd64.deb";
+    hash = "sha256-ZziTuqCuJcgRBLeF8ipXhI1vUmzfckvaEtA5BiYuCnA=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/opt/scalar $out/bin
+
+    cp -r opt/Scalar/* $out/opt/scalar/
+
+    makeWrapper ${electron}/bin/electron $out/bin/scalar \
```

```suggestion
    makeWrapper ${lig.getExe electron} $out/bin/scalar \
```
To avoid hard-coding the path, and have simple fail-safe checks (alternatively `lig.getExe' electron "electron"`